docs/quickstart/README.md

```diff
@@ -0,0 +1,298 @@
+    KernelStamper(result, kern).stampKernelDouble(re.mapToGrid(pointFeature.geom), pointFeature.data)
+    result
+  }
+
+  import geotrellis.raster.mapalgebra.local.LocalTileBinaryOp
+
+  object Adder extends LocalTileBinaryOp {
+    def combine(z1: Int, z2: Int) = {
+      if (isNoData(z1)) {
+        z2
+      } else if (isNoData(z2)) {
+        z1
+      } else {
+        z1 + z2
+      }
+    }
+    def combine(r1: Double, r2:Double) = {
+      if (isNoData(r1)) {
+        r2
+      } else if (isNoData(r2)) {
+        r1
+      } else {
+        r1 + r2
+      }
+    }
+  }
+
+  def sumTiles(t1: MutableArrayTile, t2: MutableArrayTile): MutableArrayTile = { 
+    Adder(t1, t2).asInstanceOf[MutableArrayTile]
+  }
+```
+
+Note that we require a custom Adder implementation because the built-in tile summation operator returns `NODATA` if either of the cells being added contain a `NODATA` value themselves.
+
+Creating the desired result is then a matter of the following series of operations on `pointRdd`:
+```scala
+  val tileRdd: RDD[(SpatialKey, Tile)] =
+    pointRdd
+      .flatMap(ptfToSpatialKey)
+      .mapPartitions({ partition =>
+        partition.map { case (spatialKey, pointFeature) =>
+          (spatialKey, (spatialKey, pointFeature))
+        }
+      }, preservesPartitioning = true)
+      .aggregateByKey(ArrayTile.empty(DoubleCellType, ld.tileCols, ld.tileRows))(stampPointFeature, sumTiles)
+      .mapValues{ tile: MutableArrayTile => tile.asInstanceOf[Tile] }
+```
+The `mapPartitions` operation simply applies a transformation to an RDD without triggering any kind of shuffling operation.  Here, it is necessary to make the SpatialKey available to `stampPointFeature` so that it can properly determine the pixel location in the corresponding tile.
+
+We would be finished here, except that RDDs inside GeoTrellis are required to carry along a Metadata object that describes the context of the RDD.  This is created like so:
+```scala
+  val metadata = TileLayerMetadata(DoubleCellType,
+                                   ld,
+                                   ld.extent,
+                                   LatLng,
+                                   KeyBounds(SpatialKey(0,0),
+                                             SpatialKey(ld.layoutCols-1,
+                                                        ld.layoutRows-1)))
+```
+To combine the RDD and the metadata, we write `val resultRdd = ContextRDD(tileRdd, metadata)`.
+
+This resulting RDD is essentially the object of interest, though it is possible to write `resultRDD.stitch` to produce a single merged tile.  In the general case, however, the RDD may cover an area so large and in sufficient resolution that the result of stitching would be too large for working memory.  In that case, it would be necessary to write subtiles off to disk and stitch them using some external program like GDAL, or a custom application would need to be written to make use of the tiles that were produced.
```

Comment:
`In that case, ...`

Instead of mentioning building giant rasters w/ GDAL, we should mention that in that case, we generally save off layers to one of our supported distributed backends (HDFS, S3, Accumulo, etc) and work with the tiles in a distributed way, or pull tiles out individually (for example, when serving on a web map).